platformio.ini

```diff
@@ -11,7 +11,7 @@
 [common]
 platform = espressif8266
 framework = arduino
-#board_f_cpu = 160000000L
+board_f_cpu = 160000000L
```

Comment:
I ran in some issues with syslog & websockets log enabled.

Reply:
https://github.com/puuu/MQTT433gateway/pull/21#issuecomment-351597783 :
> For the "single byte" thing: At the moment its of for me as it only ever happens if a client is actually listening. Also the selectable log levels should leave the overhead fairly low. 

Every single byte calls `log()` and this will always [iterate trough all handler](https://github.com/janLo/ArduinoSimpleLogging/blob/master/src/ArduinoSimpleLogging.cpp#L3). This is a lot of overhead for a `F(..)` string.

Better try a buffer in front of your logging, than increasing the cpu clock.

Reply:
I tested with 80 MHz and did not hit any problems. I am not willing to change the cpu clock with this pull request. If you see a need for this, please a open new PR.

Reply:
Regarding to the `F(..)` String: There is already a lot of arduino-abstraction around plus you have the time it needs to read from the SPI flash.

I especially have problems using the raw-logging. Here in my place the 433MHz band is sometimes so busy that it does not get enough time to actually decode protocols when the Raw-Log is on. For your power-concerns: All I found on the internet was, that it uses less than 15mA more at peak with the faster setting. Thats less than your LED needs. To compare: WIth full wifi-usage it can use more than 200mA.

Reply:
During Christmas I will have the possibility to check the load in a more noise environment.

Reply:
I tested with 80 MHz and did not hit any problems. Please change to 80 MHz and open a new issue/pr, as this pr is to long.